rails/Gemfile

```diff
@@ -0,0 +1,10 @@
+source 'https://rubygems.org'
+
+gem 'benchmark-ips', github: 'kirs/benchmark-ips', branch: 'quiet-mode'
+gem 'rails', path: '/rails'
```

Comment:
The ref is set by checkout out to commit here: https://github.com/ruby-bench/ruby-bench-docker/pull/7/files#diff-64e7670fdbf2e48f32d47a260e05e2f3R16
I couldn't find a better solution here.